ccpp/suites/suite_FV3_GFS_v15p2_couplednsst.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we still need a GFS v15p2 coupled nsst suite, given that we are moving all tests over to GFS v16 ?

Reply:
In fact, this suite isn't used in the regression tests - remove please.